src/pages/guide/integration/attributes/data-fixture.md

```diff
@@ -186,6 +180,7 @@ Data Fixture is a PHP class that implements `Magento\TestFramework\Fixture\DataF
 1. Data Fixture MUST NOT depend on another fixture.
 1. Data Fixture SHOULD be implemented using service APIs.
 1. Data Fixture SHOULD have dynamic default data to allow generating unique fixtures.
+1. Validation should be handled by the service class and not in the data fixture class.
```

Comment:
I think we should rephrase this to : Data Fixture MUST NOT handle input validation. Such validation should be handled by the service class.

Reply:
Corrected. I also added the scope edits.